### PR TITLE
Add automatic proofreading

### PR DESCRIPTION
**Pull request by Aryan-Singla**

### CODE REVIEW

File: doc/ref_arch/kubernetes/.github/workflows/review-docs.yml
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/.github/workflows/review-docs.yml:1]: This file should go to the `.github/workflows` folder from the **root** of the repo.

File: .vale/.vale-config/2-Hugo.ini
Comment (CsatariGergely) [inline .vale/.vale-config/2-Hugo.ini:1]: This file is not needed.

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_arch/kubernetes/.github/workflows/review-docs.yml
Comment (Aryan-Singla) [reply doc/ref_arch/kubernetes/.github/workflows/review-docs.yml:1]: Updated

File: .vale/.vale-config/2-Hugo.ini
Comment (Aryan-Singla) [reply .vale/.vale-config/2-Hugo.ini:1]: Removed

File: .github/workflows/review-docs.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: Review docs
+
+on:
+  pull_request:
+    paths:
```
Comment (CsatariGergely) [inline .github/workflows/review-docs.yml:6]:
```suggestion
```

File: .github/workflows/review-docs.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      - "ospo-book/content/en/*.md"
```
Comment (CsatariGergely) [inline .github/workflows/review-docs.yml:7]:
```suggestion
```

File: .github/workflows/review-docs.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+          BASEURL: https://github.com/errata-ai/vale/releases/download
+          VERSION: 3.3.0
+        run: |
+          mkdir -p .vale/bin && cd .vale/bin
+          wget -O vale.tar.gz --quiet "${BASEURL}/v${VERSION}/vale_${VERSION}_Linux_64-bit.tar.gz"
+          tar xzf vale.tar.gz
+          pwd >> "$GITHUB_PATH"
+      - name: Run vale
+        run: |
+          vale --output=".vale.tmpl" --no-exit ospo-book/content/en
```
Comment (CsatariGergely) [inline .github/workflows/review-docs.yml:27]:
```suggestion
          vale --output=".vale.tmpl" --no-exit .
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Review by CsatariGergely — APPROVED** [review]

**Review by CsatariGergely — APPROVED** [review]

**Comment by CsatariGergely** [comment]
I think this is as good as this can get. 
We still need to find a way to figure out how to make `github-pr-check` work. We are working on the same issue in Nephio also (https://github.com/nephio-project/docs/pull/172)

File: doc/ref_arch/kubernetes/chapters/chapter01.rst [hunk 4]
```diff
@@ -604,7 +604,7 @@ Abbreviations
    * - VTEP
      - VXLAN Tunnel End Point
    * - VXLAN
-     - Virtual Extensible LAN
+     - Virtual Extendable LAN
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter01.rst:607]:
```suggestion
     - Virtual eXtensible LAN
```
https://en.wikipedia.org/wiki/Virtual_Extensible_LAN

File: doc/ref_arch/kubernetes/chapters/chapter04.rst [hunk 5]
```diff
@@ -199,7 +199,7 @@ For a Host OS to be compliant with this Reference Architecture, it must meet the
      - Linux kernel version
      - A version of the Linux kernel that is compatible with container runtimes and kubeadm - this has been chosen as
        the baseline because kubeadm is focused on installing and managing the lifecycle of Kubernetes and nothing else,
-       hence it is easily integrated into higher-level and more complete tooling for the full lifecycle management of
+       hence it is easily integrated into higher-level and is not comparable tooling for the full lifecycle management of
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter04.rst:202]:
```suggestion
       hence it is easily integrated into higher-level tooling for the full lifecycle management of
```

**Review by rgstori — APPROVED** [review]

**Pull request merged**